CODE/logicytics/__init__.py

```
@@ -1,3 +1,29 @@
+from logicytics.Logger import Log
+from logicytics.Checks import Check
+from logicytics.FileManagement import FileManagement
+from logicytics.Flag import Flag
+import functools
 
-DEBUG, VERSION, CURRENT_FILES = Get.config_data()
+
+def deprecated(removal_version: str, reason: str) -> callable:
+    def decorator(func: callable) -> callable:
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs) -> callable:
```

Ruff (0.8.0)

Missing type annotation for `*args`

(ANN002)